rules/rules.yaml

```diff
@@ -0,0 +1,213 @@
+# This file comes from an adaptation of rules present in
+# https://github.com/webrecorder/wabac.js/blame/main/src/fuzzymatcher.js
+#
+# Syncing rules is done manually, based on expert knowledge, especially because in
+# scraperlib we are not really fuzzy matching (searching the best entry among existing
+# ones) but just rewriting to proper path.
+#
+# This file is in sync with content at commit 879018d5b96962df82340a9a57570bbc0fc67815
```

Comment:
This is a commit of wabac.js since we are speaking of syncing with this repo. I will update the comment to put the whole link to https://github.com/webrecorder/wabac.js/commit/879018d5b96962df82340a9a57570bbc0fc67815 so that it is even more obvious